### ISSUE TITLE
Display order of fields

### ISSUE THREAD

**Issue by schmikloi** [kartoza/django-bims#4359]

**Comment by schmikloi** [comment]
Order in the overview table

![Image](https://github.com/user-attachments/assets/cb20519a-120d-4607-8368-ded455f83dfd)
it should be Taxon Name, Family, Genus. Species, Author(s), Status, Accepted Taxon, Rank, Biogeographic Distribution (please note the **typo** in here!), Environmental Information

Why is there only the species name in the "Taxon Name" column for all the synonyms?

**Comment by schmikloi** [comment]
tab in the edit mode
![Image](https://github.com/user-attachments/assets/2abd243d-6fa9-4a00-abba-bb7d434fba8c)


- why does the taxon name here only display the species-part (i.e. virens)?
- order is ok for the rest

**Comment by dimasciput** [comment]
@schmikloi 

> Why is there only the species name in the "Taxon Name" column for all the synonyms?

There's a bug with this, I will fix it.

![image](https://github.com/user-attachments/assets/ec3f73e3-8e29-416d-8ea1-d82ee508ee49)

> why does the taxon name here only display the species-part (i.e. virens)?

I think it's because you can't edit the genus name here, if you want to edit the genus name you need to click the edit parent. 
cc @helendallas

**Comment by helendallas** [comment]
It is a. bit confusing to only have species part of taxon name here...not sure if there is a solution?

Maybe It shouldn't read Taxon name but rather the level you are editing?

Genus?
Species?
Subspecies?

The Taxon is the same as what we always use. the combination

**Comment by schmikloi** [comment]
To my mind it could be moved down under the Genus line.
And it should be called "Species".
And what is called Species now should then be "Taxon name".

**Comment by schmikloi** [comment]
@helendallas 
to my understanding the genus name is "Afronemoura"
the species name is "spinulata"
and the taxon name is "Afroneumura spinulata"

really correct would be the following terminology: The first part of the name – the generic name – identifies the [genus] to which the species belongs, whereas the second part – the specific name or specific epithet – distinguishes the species within the genus...

that being said we could leave the order as is, just show the entire Taxon Name in the respective field, i.e. "Afroneumura spinulata" and not only "spinulata"

what do you think @helendallas ?

**Comment by helendallas** [comment]
Agreed - That's fine. For me its important that Taxon name includes the full name as we have in all other BIMS versions, as you have indicated above

**Comment by dimasciput** [comment]
This has been updated

**Comment by helendallas** [comment]
@dimasciput Some species are now coming through with genus and species. I suspect thins to do with old data and or data uploaded and edited etc.

@schmikloi I would suggest that we deploy all to production before Dimas goes on leave so that you cna start with a Clea slate and then set up users, experts and then start preparing and uploading cleaned taxon lists.

I am going to close this ticket as completed

**Comment by helendallas** [comment]
Unfortunately column header is not matching information show.  It is correct on FADA but incorrect on others:

FBIS
![Image](https://github.com/user-attachments/assets/11e6e6a7-ffc8-4029-aa9c-4fff42a3f23c)


SANParks



![Image](https://github.com/user-attachments/assets/05baa8b4-5101-4d34-8992-9f45e1545b8f)

Kafue

![Image](https://github.com/user-attachments/assets/4d010b33-2aeb-4a81-8ce6-69f2b4207600)

**Comment by schmikloi** [comment]
two issues from this thread are still open; I will move them into new tickets

**Comment by schmikloi** [comment]
Dear sender,

Thank you for your email. I am currently out of office and will return on February 10th. I may not check emails during this time but will respond as soon as possible after my return.

Thank you for your understanding.

Best regards,
Astrid
